packages/currency/src/index.js

```diff
@@ -48,6 +49,24 @@ const CurrencyFactory = ( currencySetting ) => {
 		return sprintf( priceFormat, symbol, formattedNumber );
 	}
 
+	/**
+	 * Formats money value.
+	 *
+	 * @deprecated
+	 *
+	 * @param   {number|string} number number to format
+	 * @return {?string} A formatted string.
+	 */
+	function formatCurrency( number ) {
+		deprecated( 'Currency().formatCurrency', {
+			version: '4.3.0',
```

Comment:
When are we planning to fully _remove_ `formatCurrency()`? This produces the message:

> Currency().formatCurrency is deprecated and will be removed from WooCommerce in version 4.3.0

Considering that's a past version, the message is confusing.

Reply:
Going off of the [Woo Core Docs](https://github.com/woocommerce/woocommerce/wiki/Deprecation-in-core) we don't have a real guideline about choosing which version a method will be finally removed in... to play it safe, I feel like we should say it will be removed in the next breaking release ( 5.0.0 ) - and we can standardize on that then easily search the code base for said deprecation notices prior to that release and cleanup the old methods then. Does that sound reasonable to you all?

Reply:
Good call team, thanks!